apps/vr-tests/src/stories/react-charting/VerticalBarChart.stories.tsx

```diff
@@ -0,0 +1,404 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { TestWrapperDecorator } from '../../utilities/TestWrapperDecorator';
+import { Steps, StoryWright } from 'storywright';
+import {
```

Comment:
For some cases of vertical bar, we can set hideLabels = true

Reply:
ok will do